docs/source/new-docs/contribute/coding_standards.rst

```diff
@@ -29,6 +29,300 @@ right in the case of a conflict with this document.
 External Style Standards
 ------------------------
 
+Numpy Docstrings
+^^^^^^^^^^^^^^^^
+
+Full information here: `Docstring Standard <https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>`_
+
+Common mishaps:
+"""""""""""""""
+
+The first docstring line should be a "one-line summary that **does not use variable
+names or the function name**"
+
+Classes
```

Comment:
Was this example just pulled from numpy, or did you write it from scratch?  If it's pulled from numpy should it just be a link?  Or is this a somewhat geoips specific flavor of numpy docstring formatting as an example?

Reply:
They're from scratch, and are a result of a request for a "copy-and-paste" example in the docs for what a good numpy docstring looks like